zep_python/document/collections.py

```diff
@@ -640,7 +640,7 @@ def search_return_query_vector(
 
     def search(
         self,
-        text: Optional[str] = None,
+        text: Optional[Union[str, Dict[str, str]]] = None,
```

Comment:
why is this necessary?

Reply:
Updated rag example to extract string from the dict before passing it to the retriever, my bad